### PR TITLE
add proxy.zimplylost.net

### PR DESCRIPTION
**Pull request by DarkZTVDE**
## Type of change

- [✅] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [✅] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [✅] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [✅] I have created the manifest.json with the required values

## Further comments
Minecraft Minigames Server with Bowling, Minigolf, Smash and Tower Defense

### CODE REVIEW

File: minecraft_servers/zimplylost/manifest.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      "color": "#0095B0"
+    },
+    "towerdefense": {
+      "name": "Tower Defense",
+      "color": "#0095B0"
+    }
+  },
+  "discord": {
+    "server_id": 781614288929554453,
+    "rename_to_minecraft_name": true
```
Comment (maaaathis) [inline minecraft_servers/zimplylost/manifest.json:41]: Reserved for partners only; change to `false`

File: minecraft_servers/zimplylost/manifest.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "server_wildcards": [
+    "mc.zimplylost.net",
+	"proxy.zimplylost.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://zimplylost.net",
```
Comment (maaaathis) [inline minecraft_servers/zimplylost/manifest.json:14]: image

**Review by maaaathis — CHANGES_REQUESTED** [review]
Your domain `zimplylost.net` is expired.
Google Chrome 2024-05-29 07 32 13

**Comment by DarkZTVDE** [comment]
All Done, the Domain ZimplyLost.net is now online :)

File: minecraft_servers/zimplylost/manifest.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  "server_wildcards": [
+    "mc.zimplylost.net",
+	"proxy.zimplylost.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://www.zimplylost.net",
```
Comment (DoJapHD) [inline minecraft_servers/zimplylost/manifest.json:14]: Website doesn't have valid SSL certificate

File: minecraft_servers/zimplylost/manifest.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    "web_shop": "https://shop.zimplylost.net",
```
Comment (DoJapHD) [inline minecraft_servers/zimplylost/manifest.json:15]: Doesn't exist

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server is in beta; Reopen PR once server is publicly available

**Pull request closed without merging**